fe2-android/app/src/main/java/com/github/dedis/student20_pop/model/Lao.java

```diff
@@ -45,11 +45,11 @@ public void updateRollCall(String prevId, RollCall rollCall) {
   }
 
   public void updateElection(String prevId, Election election) {
-    if (elections.containsKey(prevId)) {
-      elections.remove(prevId);
-    }
-    String newId = election.getId();
-    elections.put(newId, election);
```

Comment:
Why has this check been deleted?

Reply:
If I'm not mistaking, when you use .put on a map, the old value will be replaced by the new one, or added if no such mapping exists. As the id of the election doesn't change, the prevId parameter will always correspond to election.getId() (since we simply want to update an election), so it seemed like this check was useless.